### DISCUSSION TITLE
Error:Something went wrong

### DISCUSSION THREAD

**Discussion by igeogan0402** [AntlerVC/firetable#927] [Support / Q&A] [answered]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Rowy Run version**
If applicable, share the Rowy Run version from your project settings.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by BeeBombshell** [comment, marked as answer]
Hey @igeogan0402. It's a little hard to understand what exactly the issue is. Kindly share a screenshot or a small video demonstrating the same for further help from the dev team.
